StyleCop.Analyzers/StyleCop.Analyzers/Helpers/TaskEx.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace StyleCop.Analyzers.Helpers
+{
+    using System.Threading.Tasks;
+
+    internal static class TaskEx
```

Comment:
:exclamation: Let's call this [`SpecializedTasks`](https://github.com/dotnet/roslyn/blob/master/src/Workspaces/Core/Portable/Utilities/SpecializedTasks.cs) instead.